src/compiler/commandLineParser.ts

```diff
@@ -583,6 +584,12 @@ namespace ts {
             category: Diagnostics.Experimental_Options,
             description: Diagnostics.Enables_experimental_support_for_emitting_type_metadata_for_decorators
         },
+        {
+            name: "experimentalBigInt",
+            type: "boolean",
+            category: Diagnostics.Experimental_Options,
+            description: Diagnostics.Enables_experimental_support_for_ESNext_BigInt_expressions

```

Comment:
Handle bigint as key in ParsedJson to report error: `convertPropertyValueToJson` add test as well.

Reply:
Handle type `JsonMinusNumericLiteral`